unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -1294,13 +1326,15 @@ who never yet saw the witchery of the whorish spirit from the life,
 and how he himself hath been bewitched and cozened with the false, instead of the true?
 As for instance:
 
-Prayer; that is generally taken for a means: "`Ask, and it shall be given you; seek,
+[.discourse-part]
+_Prayer;_ that is generally taken for a means: "`Ask, and it shall be given you; seek,
```

Comment:
It seemed to me that italicizing this was right to distinguish it as discourse? But I wasn't sure.

Reply:
Yeah, good thought. I waffled back and forth a few times, but ultimately I think let's get rid of the italics. I like labeling it as a discourse-part, because it's sort of a proposition answered below, but I think we can just have no italics here and it feels slightly more correct.